components/cycles/CreateCycle.jsx

```diff
@@ -3,9 +3,75 @@ import { AntDesign } from '@expo/vector-icons';
 import * as React from 'react';
 import { View } from 'react-native';
 import styled from 'styled-components/native';
+import FullCycleOrder from './FullCycleOrder';
 import FinishButton from '../utils/FinishButton';
 import PlusButton from '../utils/PlusButton';
 
+const workouts = [
```

Comment:
**This needs to be state**

Reply:
I thought this was cole's part since I didn't make this file but I don't mind doing it

Reply:
yeah can you do this part please

Reply:
yep